orchagent/flexcounterorch.cpp

```diff
@@ -144,3 +158,13 @@ void FlexCounterOrch::doTask(Consumer &consumer)
         consumer.m_toSync.erase(it++);
     }
 }
+
+bool FlexCounterOrch::getPortCountersState()
+    {
+        return m_port_counter_enabled;
+    }
```

Comment:
Could you follow the indentation convention of nearby code?

Reply:
Fixed.